zqd/search/search.go

```diff
@@ -75,7 +75,11 @@ func Copy(ctx context.Context, w zbuf.Writer, r zbuf.Reader, prog string) error
 	if err != nil {
 		return err
 	}
-	d := zdriver.New(w)
+	if len(w) == 1 {
+		d := zdriver.New(w[0])
+		return d.Run(mux)
+	}
+	d := zdriver.New(w...)
```

Comment:
I don't think you need the length check here. Shorten to:
```suggestion
	d := zdriver.New(w...)
```

Reply:
oops... leftover from 2f55965dede9c372808b3a9e7dda869741f43ff1